velox/functions/prestosql/tests/MapTest.cpp

```diff
@@ -99,6 +99,26 @@ TEST_F(MapTest, noNulls) {
   assertEqualVectors(expectedMap, result);
 }
 
+TEST_F(MapTest, emptyMap) {
+  const auto numRows = 1'000;
+  // We use unknown types here because no way to specify the output type for the
+  // map() call.
+  const TypePtr emptyMapType = MAP(UNKNOWN(), UNKNOWN());
```

Comment:
emptyMapType variable is not used; please, remove.

Reply:
Damn, doing multiple things at once is really unproductive. :)